<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>booque</class>
 <widget class="QWidget" name="booque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Image/img/logo.png</normalon>
   </iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QFrame" name="navbar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>110</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="home_btn">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>110</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#home_btn:hover {
	background-color: rgba(255, 255, 255, 0);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Image/img/logo_full.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>250</width>
          <height>65</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="profile">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="prof">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>50</width>
             <height>50</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
          </widget>
          <widget class="QPushButton" name="prof_btn">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>50</width>
             <height>50</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">#prof_btn:hover {
	background-color: rgba(255, 255, 255, 0);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>35</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QStackedWidget" name="app_panel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1200</width>
       <height>690</height>
      </size>
     </property>
     <widget class="Home" name="home_page"/>
     <widget class="Profile" name="profile_page"/>
     <widget class="Edit" name="edit_page"/>
     <widget class="Library" name="library_page"/>
     <widget class="Request" name="request_page"/>
     <widget class="Setting" name="setting_page"/>
     <widget class="Book" name="book_page"/>
     <widget class="Chart" name="chart_page"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Home</class>
   <extends>QWidget</extends>
   <header>gui/home</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Profile</class>
   <extends>QWidget</extends>
   <header>gui/profile</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Edit</class>
   <extends>QWidget</extends>
   <header>gui/edit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Setting</class>
   <extends>QWidget</extends>
   <header>gui/setting</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Request</class>
   <extends>QWidget</extends>
   <header>gui/request</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Library</class>
   <extends>QWidget</extends>
   <header>gui/library</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Book</class>
   <extends>QWidget</extends>
   <header>gui/book</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Chart</class>
   <extends>QWidget</extends>
   <header>gui/chart</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rsrc.qrc"/>
 </resources>
 <connections/>
</ui>
